modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -18,17 +19,80 @@ As an Operator author, your Operator must meet additional requirements to run pr
 // TODO: Include more info w/ better steps on how to do this:
 //* You must understand the {product-title} proxy configuration.
 
-For the CSV requirements, you can make the following changes as the Operator author.
+For the CSV requirements, you can make the following changes as the Operator author. The following procedure uses the Memcached Operator created in the Operator SDK tutorials for Go, Ansible, and Helm.
 
 .Prerequisites
 
 * An Operator project with a CSV.
 
 .Procedure
 
-. Use SHA references to related images in two places in the CSV for your Operator:
+. Replace hard-coded image references with environment variables in the following files:
 
-.. Update `spec.relatedImages`:
+** For Go- and Ansible-based Operator projects, update the `roles/memcached/tasks/main.yml` file:
++
+[source,diff]
+----
+...
+spec:
+  containers:
+  - name: memcached
+    command:
+    - memcached
+    - -m=64
+    - -o
+    - modern
+    - -v
+-   image: "docker.io/memcached:1.4.36-alpine"
++   image: "{{ lookup('env', 'RELATED_IMAGE_MEMCACHED') }}"
+    ports:
+      - containerPort: 11211
+...
+----
+
+** For Helm-based Operator projects, update the `helm-charts/memchached/values.yaml` file:
++
+[source,diff]
+----
+  ## Memcached image and tag
+  ## ref: https://hub.docker.com/r/library/memcached/tags/
+  ##
+- image: memcached:1.5.20
++ image: "{{ lookup('env', 'RELATED_IMAGE_MEMCACHED') }}"
+...
+----
+
+.Give the images and tags used by the Operator by using an environment variable in the `config/manager/manager.yaml` file:
++
+[source,yaml]
+----
+...
+spec:
+  ...
+    spec:
+      ...
+      containers:
+      - command:
+        - /manager
+        ...
+        env:
+        - name: RELATED_IMAGE_MEMCACHED
+          value: "quay.io/example.com/image:0.0.1"
+----
+
+. Use the `make bundle` command and set `USE_IMAGE_DIGESTS` to `true` to automatically list any related images that your Operator might require and reference each image by a digest (SHA), not a tag:
++
+[source,terminal]
+----
+$ make bundle USE_IMAGE_DIGESTS=true
```

Comment:
Should add one step before "make bundle USE_IMAGE_DIGESTS=true" because of bug [2047552](https://bugzilla.redhat.com/show_bug.cgi?id=2047552)

Update the Makefile
" $(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)" 

to be 

"$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_GEN_FLAGS)"

Reply:
Others LGTM